dpnp/backend/queue_sycl.cpp

```diff
@@ -83,7 +84,7 @@ void backend_sycl::backend_sycl_queue_init(QueueOptions selector)
 
     if (queue)
     {
-        backend_sycl::destroy();
+        backend_sycl::destroy_queue();
```

Comment:
revert back the function name

Reply:
Do you want me destroy queue every time, when I am init new engine? I think this is not in our case